api/api.go

```diff
@@ -370,9 +370,15 @@ func (api *API) getEntitiesHandler(c echo.Context) error {
 
 	// Query Filter
 	f := etre.QueryFilter{}
-	csv := c.QueryParam("labels")
-	if csv != "" {
-		f.ReturnLabels = strings.Split(csv, ",")
+	queryParam := c.QueryParams() // ?x=1&y=2&z -> https://godoc.org/net/url#Values
+	if csv, ok := queryParam["labels"]; ok {
+		f.ReturnLabels = strings.Split(csv[0], ",")
+	}
+	if _, ok := queryParam["distinct"]; ok {
+		f.Distinct = true
+	}
+	if f.Distinct && len(f.ReturnLabels) > 1 {
+		return readError(ErrInvalidQuery.New("distinct requires only 1 return label but %d specified: %v", len(f.ReturnLabels), f.ReturnLabels))
```

Comment:
I can think of at least one example where we want to return multiple labels, but we want one of them to be distinct. In that case, something like `es entity.label1,label2 --distinct label2` could be useful

Reply:
Yeah, unfortunately MongoDB doesn't support that.

Reply:
i.e. [distinct()](https://docs.mongodb.com/manual/reference/method/db.collection.distinct/) doesn't support it. I bet there's another way to do it, though.